go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/dapr/components-contrib
 
-go 1.12
+go 1.13
```

Comment:
Upgrade to 1.13 as per dapr/dapr repo, we can keep it as 1.12 if you guys are having any concern about compatibility.

Reply:
We use 1.13.3 in CI. it should be fine. cc/ @yaron2

Reply:
Should be ok